gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/domain_service/CreateApiDomainServiceImpl.java

```diff
@@ -88,6 +105,10 @@ public ApiWithFlows create(Api api, AuditInfo auditInfo) {
 
         flowCrudService.saveApiFlows(api.getId(), api.getApiDefinitionV4().getFlows());
 
+        if (isApiReviewEnabled(auditInfo.organizationId(), auditInfo.environmentId())) {
```

Comment:
Once again, I"m wondering if it could be dangerous in the future to rely on audit information to retrieve org & env information 🤔